github/repo_delete.go

```diff
@@ -5,7 +5,7 @@ import (
 	"github.com/g4s8/gitstrap/context"
 	"log"
 )
-
+// @todo #44:30min Remove unused variable
```

Comment:
@GnusinPavel what's this `@todo` for? Looks like it's not related to origin task

Reply:
there is a hint about unused variable, so I decided to put a puzzle

Reply:
Should I remove this puzzle?

Reply:
@GnusinPavel if it's just a single line to remove, then you can do it right in PR. But if requires some bigger refactoring, then you can submit a separate ticket

Reply:
fixed, yes it's just a single line